spring-boot/admin/src/main/java/eu/coatrack/admin/service/GatewayHealthMonitorService.java

```diff
@@ -40,53 +39,56 @@
 @Service
 public class GatewayHealthMonitorService {
 
-    @Value("${ygg.gateway-health-monitor.warning.threshold.minutes}")
-    private int gatewayHealthWarningThresholdInMinutes;
+    public GatewayHealthMonitorService(@Value("${ygg.gateway-health-monitor.warning.threshold.minutes}") int gatewayHealthWarningThresholdInMinutes,
```

Comment:
Constructor should usually be placed below the fields.

Reply:
Done and lesson learned ;D